lib/platform/bitbucket/index.ts

```diff
@@ -332,6 +332,7 @@ interface PrResponse {
       name: string;
     };
   };
+  reviewers: Array<any>;
```

Comment:
Can we be more specific here? Looks like `reviewers: {username: string}[]` might be more appropriate.

Reply:
The type returned here is a bit more complicated than that. It can return different fields, some that are populated and some that are not. I felt that a simple solution would be to put `any` here since we don't really care what the values are, we just immediately return them to Bitbucket in the very next call.

Here's are links to the relevant API docs: 
- Pull request response: https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Bworkspace%7D/%7Brepo_slug%7D/pullrequests/%7Bpull_request_id%7D
- User/account object: https://developer.atlassian.com/bitbucket/api/2/reference/resource/user

```
reviewers0..∞

The list of users that were added as reviewers on this pull request when it was created. For performance reasons, the API only includes this list on a pull request's self URL.
all of
object

Base type for most resource objects. It defines the common type element that identifies an object's type. It also identifies the element as Swagger's discriminator.
typestring

An account object.
...
```

Reply:
If we don't care, we should use `unknown` instead of `any`